src/client.rs

```diff
@@ -1058,17 +1148,38 @@ mod tests {
     #[meilisearch_test]
     async fn test_list_all_indexes(client: Client, index: Index) {
         let all_indexes = client.list_all_indexes().await.unwrap();
-        assert!(all_indexes.len() > 0);
-        assert!(all_indexes.iter().any(|idx| idx.uid == index.uid));
+
+        assert_eq!(all_indexes.limit, 20);
+        assert_eq!(all_indexes.offset, 0);
+        assert!(all_indexes.results.iter().any(|idx| idx.uid == index.uid));
```

Comment:
Actually, this can fail now.
If there is more than 20 index in the meilisearch instance, we may not find our own index :unamused: 

I guess we have no choice but to remove this check + the `index` parameter.